build.gradle

```diff
@@ -46,7 +46,7 @@ dependencies {
 
     implementation "com.google.protobuf:protobuf-java:$protobufVersion"
 
-    //implementation "javax.annotation:javax.annotation-api:1.3.2"
+    implementation "javax.annotation:javax.annotation-api:1.3.2"
```

Comment:
I was unable to get the project to build without this line uncommented. Was I possibly missing a setup step?